views/update/update-trust-tell-us-about-individual-bo.html

```diff
@@ -0,0 +1,15 @@
+{% extends "update-layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
```

Comment:
Is this title correct?

Reply:
Fixed

Reply:
Ah right, that's how it was written when I started and is present on main. To remain consistent I'll put the string back in

Reply:
Can you please ask Matthew what the content should be? Then the pattern for update is not to use page params but it have it in the HTML.